.github/workflows/build.yml

```diff
@@ -105,7 +105,7 @@ jobs:
         os: [ubuntu-latest, windows-latest, macos-latest]
     env:
       # Set up wheels matrix.  This is CPython 3.6--3.9 for all OS targets.
-      CIBW_BUILD: "cp3[6-9]-*"
+      CIBW_BUILD: "cp3{6,7,8,9}-*"
```

Comment:
Do you want to build with 3.10 as well?

Reply:
I explicitly didn't add 3.10 in this PR, because I wanted to add CI test builds first, and that turned out to be a bit non-trivial (because SciPy for 3.10 wasn't in the main conda repository yet). So I'm leaving it for a follow up PR.